front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -30,29 +30,27 @@ export default function SpeedLimitByTagSelector({
   );
 
   if (!speedLimitsTagsList.length) return null;
+
   return (
     <div className="osrd-config-item mb-2">
       <div
         className={cx('osrd-config-item-container', {
           'd-flex align-items-center gap-10': condensed,
         })}
       >
-        <img width="32px" src={icon} alt="speedometer" />
-        <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
           disabled={disabled}
-          withSearch
-          dataTestId="speed-limit-by-tag-selector"
-          value={speedLimitByTag || t('noSpeedLimitByTag').toString()}
```

Comment:
I think we loose the ability to clear the speed limit by tag selection?

Could be fixed with something like `placeholder={t('noSpeedLimitByTag')}`.

Reply:
To expand on this a bit: before a user could select a speed limit by tag and then unselect it. Now there is no way to unselect it. Once selected, the user is stuck with the speed limit by tag.